maps/site53/items/clothing/solgov-accessory.dm

```diff
@@ -116,45 +111,41 @@ armbands
 /*****************
 armour attachments
 *****************/
-/obj/item/clothing/accessory/armor/tag/solgov
-	name = "\improper SCG Flag"
-	desc = "An emblem depicting the Sol Central Government's flag."
+/obj/item/clothing/accessory/armor/tag/goc
+	name = "\improper UNGOC Flag"
+	desc = "An emblem depicting the UNGOC's flag."
 	icon_override = 'maps/torch/icons/obj/solgov-accessory.dmi'
 	icon = 'maps/torch/icons/obj/solgov-accessory.dmi'
 	accessory_icons = list(slot_tie_str = 'maps/torch/icons/mob/solgov-accessory.dmi', slot_w_uniform_str = 'maps/torch/icons/mob/solgov-accessory.dmi', slot_wear_suit_str = 'maps/torch/icons/mob/solgov-accessory.dmi')
 	icon_state = "solflag"
 	slot = ACCESSORY_SLOT_ARMOR_M
 
-/obj/item/clothing/accessory/armor/tag/solgov/ec
-	name = "\improper Expeditionary Corps crest"
-	desc = "An emblem depicting the crest of the SCG Expeditionary Corps."
-	icon_state = "ecflag"
-
-/obj/item/clothing/accessory/armor/tag/solgov/sec
+/obj/item/clothing/accessory/armor/tag/goc/sec
 	name = "\improper GUARD tag"
 	desc = "An armor tag with the word GUARD printed in silver lettering on it."
 	icon_state = "sectag"
 
-/obj/item/clothing/accessory/armor/tag/solgov/com
-	name = "\improper SCG tag"
-	desc = "An armor tag with the words SOL CENTRAL GOVERNMENT printed in gold lettering on it."
+/obj/item/clothing/accessory/armor/tag/goc/com
 	icon_state = "comtag"
 
-/obj/item/clothing/accessory/armor/tag/solgov/com/sec
-	name = "\improper POLICE tag"
-	desc = "An armor tag with the words POLICE printed in gold lettering on it."
+/obj/item/clothing/accessory/armor/tag/goc/com/goc
+	name = "\improper UNGOC tag"
+	desc = "An armor tag with the words UNITED NATIONS - GLOBAL OCCULT COALITION printed in gold lettering on it."
+/obj/item/clothing/accessory/armor/tag/goc/com/uiu
+	name = "\improper FBI tag"
+	desc = "An armor tag with the words FEDERAL BUREAU OF INVESTIGATION printed in gold lettering on it."
 
-/obj/item/clothing/accessory/armor/tag/solgov/com/guardcomm
+/obj/item/clothing/accessory/armor/tag/goc/com/guardcomm
```

Comment:
guard commander and zone commanders are pins worn by foundation staff, hence it should be foundation not goc

Reply:
fixed